presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/JsonRenderer.java

```diff
@@ -62,16 +63,26 @@ private JsonRenderedNode renderJson(PlanRepresentation plan, NodeRepresentation
         private final String type;
         private final String identifier;
         private final String details;
+        private final String verboseDetails;
         private final List<JsonRenderedNode> children;
         private final List<String> remoteSources;
 
-        public JsonRenderedNode(String id, String name, String type, String identifier, String details, List<JsonRenderedNode> children, List<String> remoteSources)
+        public JsonRenderedNode(
+                String id,
+                String name,
+                String type,
+                String identifier,
+                String details,
+                String verboseDetails,
+                List<JsonRenderedNode> children,
```

Comment:
Let's not put verbose details in JsonRenderedNode until we have a use case for it.